qcodes/tests/dataset/dataset_fixtures.py

```diff
@@ -35,6 +35,27 @@ def scalar_dataset(dataset):
     yield dataset
 
 
+@pytest.fixture
+def scalar_dataset_with_nulls(dataset):
+    """
+    A very simple dataset. A scalar is varied, and two parameters are measured
```

Comment:
i'd argue that this is not "simple" :)

This is simple:
```
1 | 25 | 46
```

This is less simple:
```
1 | 25   | NULL
1 | NULL | 42
```